webapp/src/main/java/life/qbic/views/login/LoginLayout.java

```diff
@@ -38,44 +36,45 @@ public class LoginLayout extends VerticalLayout implements HasUrlParameter<Strin
 
   private VerticalLayout contentLayout;
 
-  public PasswordField password;
+  private VerticalLayout informationContainer;
 
-  public EmailField email;
+  private VerticalLayout errorContainer;
```

Comment:
I think you can replace this with just one "notificationContainer", since it's purpose is to contain the notification. The type will be determined by the notification that is shown.

Reply:
We need to have the possibility to handle errors and information separate from each other. As the login view serves multiple purposes (login and email confirmation)